com.unity.render-pipelines.high-definition/Runtime/Deprecated.cs

```diff
@@ -0,0 +1,214 @@
+using System;
+
+namespace UnityEngine.Rendering.HighDefinition
+{
+    /// <summary>
+    /// Possible values for one element of the Local Volumetric Fog atlas.
+    /// </summary>
+    [Serializable]
+    [Obsolete("DensityVolumeResolution has been deprecated (UnityUpgradable) -> LocalVolumetricFogResolution", false)]
+    public enum DensityVolumeResolution
```

Comment:
I wonder. The scripting API will fox user code to use LocalVolumetricFogResolution instead of DensityVolumeResolution. Is there a reason to keep the content of the old deprecate DensityVolumeResolution?